docs/getting-started-v2-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De 'Bevraging Persoon' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-api)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
+
+De (resolved) OAS3 kun je hier downloaden: [openapi.yaml](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml).
+
+De [functionele documentatie](./features) vind je in [features](./features).
+
+## Implementeer de API client
+Client code kun je genereren met de "[genereervariant](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties en een code generator. Een overzicht met codegeneratoren kun je vinden op [OpenAPI.Tools](https://openapi.tools/#sdk){:target="_blank" rel="noopener"}.
+
+Deze repo bevat scripts waarmee je met [OpenAPI Generator](https://openapi-generator.tech/){:target="_blank" rel="noopener"} client code kunt genereren in JAVA, .NET (Full Framework & Core) en Python. De makkelijkste manier om de code generatie scripts te gebruiken, is door deze repo te clonen. Na het clonen kun je met `npm install` de benodigde packages installeren en kun je met npm run <script naam> één van de volgende scripts uitvoeren:
```

Comment:
Zijn de tweede en derde zin nog van toepassing? Of is dat nog een restant uit de oude situatie dat wij ook gegenereerde code opnamen in de repo?